ePBFT/src/pbft/libbyz/test/test_ledger_replay.cpp

```diff
@@ -83,31 +84,28 @@ void init_replica(std::vector<char>& service_mem)
 
```

Comment:
Please look at ledger.cpp to see how to use the existing ledger interface. We do not want pbft-tests (or any other tests) to be using a different interface to the ledger than the existing code and tests do.

Reply:
ok